src/services/config.js

```diff
@@ -146,9 +146,8 @@ class ConfigData {
     this.edge = configObj.edge;
     this.email = configObj.email || {};
     this.prompts = configObj.prompts || {};
-    // Note the historical 'projects' naming.
-    configObj.profiles = configObj.projects || [];
-    configObj.profiles.forEach((profile) => this.addProfile(profile.id, profile.accountSid, profile.region));
+    configObj.projects = configObj.projects || [];
+    configObj.projects.forEach((project) => this.addProfile(project.id, project.accountSid, project.region));
     this.setActiveProfile(configObj.activeProject);
```

Comment:
Removed comment still applies to this line for profile/project.

Reply:
Added back comments at both locations.